include/SDL3/SDL_video.h

```diff
@@ -1801,6 +1801,40 @@ extern SDL_DECLSPEC SDL_bool SDLCALL SDL_WindowHasSurface(SDL_Window *window);
  */
 extern SDL_DECLSPEC SDL_Surface *SDLCALL SDL_GetWindowSurface(SDL_Window *window);
 
+/**
+ * Toggle VSync for the window surface.
+ *
+ * When a window surface is created, vsync defaults to SDL_WINDOW_SURFACE_VSYNC_DISABLED.
+ *
+ * \param window the window
+ * \param vsync the vertical refresh sync interval, 1 to synchronize present with every vertical refresh, 2 to synchronize present with every second vertical refresh, etc., SDL_WINDOW_SURFACE_VSYNC_ADAPTIVE for late swap tearing (adaptive vsync), or SDL_WINDOW_SURFACE_VSYNC_DISABLED to disable. Not every value is supported by every driver, so you should check the return value to see whether the requested setting is supported.
```

Comment:
I'm going to move most of this text to the remarks section, but let's merge this first and then I'll clean that up.

Reply:
Cool, you probably want to move that text for SDL_SetRenderVSync() too.

Reply:
Done and done in e508d82f746644973940a2a2c576573876803427.